### PR TITLE
update FileSystemFileHandle

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Motivation

mainly for update exception section

* getFile
  * NotAllowedError step4.3.1
  * NotFoundError step4.3.2
* createWritable
  * NotAllowedError step5.3
  * NotFoundError step5.4
  * NoModificationAllowedError step5.7.1
  * AbortError step5.7.2->step4.2.2.2
* createSyncAccessHandle
  * NotAllowedError step6.3
  * InvalidStateError step6.4
  * NotFoundError step6.5
  * NoModificationAllowedError step6.8.1

### Additional details

See https://fs.spec.whatwg.org/#api-filesystemfilehandle

### CODE REVIEW

**Comment by bsmth** [comment]
Hi @skyclouds2001 - thanks a lot for the additions. I notice you have quite a few draft PRs open, are any of them ready to be taken out of draft status for review? https://github.com/mdn/content/pulls?q=is%3Aopen+is%3Apr+author%3Askyclouds2001+is%3Adraft

**Comment by skyclouds2001** [comment]
> Hi @skyclouds2001 - thanks a lot for the additions. I notice you have quite a few draft PRs open, are any of them ready to be taken out of draft status for review? https://github.com/mdn/content/pulls?q=is%3Aopen+is%3Apr+author%3Askyclouds2001+is%3Adraft

Oh, got it, I'll check as soon as possible

File: files/en-us/web/api/filesystemfilehandle/createsyncaccesshandle/index.md [hunk 1]
```diff
@@ -31,12 +31,14 @@ A {{jsxref('Promise')}} which resolves to a {{domxref('FileSystemSyncAccessHandl
 
 ### Exceptions
 
+- `NotAllowedError` {{domxref("DOMException")}}
+  - : Thrown if the {{domxref('PermissionStatus.state')}} for the handle is not `'granted'` in `readwrite` mode.
```
Comment (sideshowbarker) [inline files/en-us/web/api/filesystemfilehandle/createsyncaccesshandle/index.md:35]:
```suggestion
  - : Thrown if the {{domxref('PermissionStatus.state')}} for the handle is not `granted` in `readwrite` mode.
```

File: files/en-us/web/api/filesystemfilehandle/createwritable/index.md [hunk 2]
```diff
@@ -40,6 +40,12 @@ A {{jsxref('Promise')}} which resolves to a {{domxref('FileSystemWritableFileStr
 
 - `NotAllowedError` {{domxref("DOMException")}}
   - : Thrown if the {{domxref('PermissionStatus.state')}} for the handle is not `'granted'` in `readwrite` mode.
+- `NotFoundError` {{domxref("DOMException")}}
+  - : Thrown if current entry is not found.
+- `NoModificationAllowedError` {{domxref("DOMException")}}
+  - : Thrown if the browser is not able to acquire a lock on the file associated with the file handle.
+- `AbortError` {{domxref("DOMException")}}
+  - : Thrown if implementation-defined malware scans and safe browsing checks fails.
```
Comment (sideshowbarker) [inline files/en-us/web/api/filesystemfilehandle/createwritable/index.md:48]:
```suggestion
  - : Thrown if implementation-defined malware scans and safe-browsing checks fails.
```

File: files/en-us/web/api/filesystemfilehandle/getfile/index.md [hunk 3]
```diff
@@ -32,8 +32,9 @@ A {{jsxref('Promise')}} which resolves to a {{domxref('File')}} object.
 ### Exceptions
 
 - `NotAllowedError` {{domxref("DOMException")}}
-  - : Thrown if the {{domxref('PermissionStatus.state')}} is not `granted` in
-    read mode.
+  - : Thrown if the {{domxref('PermissionStatus.state')}} is not `'granted'` in `read` mode.
```
Comment (sideshowbarker) [inline files/en-us/web/api/filesystemfilehandle/getfile/index.md:35]:
```suggestion
  - : Thrown if the {{domxref('PermissionStatus.state')}} is not `granted` in `read` mode.
```

**Review by sideshowbarker — APPROVED** [review]

**Pull request merged**